src/views/confirmation.njk

```diff
@@ -1,31 +1,54 @@
 {% extends 'layout.njk' %}
 
 {% from 'govuk/components/panel/macro.njk' import govukPanel %}
+{% from 'govuk/components/summary-list/macro.njk' import govukSummaryList %}
 
 {% block beforeContent %}{% endblock %}
 
 {% block content %}
+  <noscript>
```

Comment:
Would be ideal to place it in `{% block bodyStart %}{% endblock %}` block so it is placed at the beginning of body. It will be called sooner as one benefit and it will mirror what we do in `{% block bodyEnd %}` as second benefit.